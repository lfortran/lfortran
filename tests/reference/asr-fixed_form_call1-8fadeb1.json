{
    "basename": "asr-fixed_form_call1-8fadeb1",
    "cmd": "lfortran --indent --fixed-form --allow-implicit-interface --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_call1.f",
    "infile_hash": "cb6674a06b92dda41650f02e8f066e0e23bc067d3d89161c62dfb256",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_call1-8fadeb1.stdout",
    "stdout_hash": "b1af60fdba250ecc6eaacb4fdfe366b923c78a741117c6111bdacc1c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}