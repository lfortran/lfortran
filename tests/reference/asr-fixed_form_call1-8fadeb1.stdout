(TranslationUnit 
    (SymbolTable
        1
        {
            idd_frm:
                (Function 
                    (SymbolTable
                        2
                        {
                            lw:
                                (Variable 
                                    2 
                                    lw 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            m:
                                (Variable 
                                    2 
                                    m 
                                    Unspecified 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            n:
                                (Variable 
                                    2 
                                    n 
                                    Unspecified 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            prinf:
                                (Function 
                                    (SymbolTable
                                        3
                                        {
                                            prinf_arg_0:
                                                (Variable 
                                                    3 
                                                    prinf_arg_0 
                                                    Unspecified 
                                                    () 
                                                    () 
                                                    Default 
                                                    (Character 1 6 () []) 
                                                    Source 
                                                    Public 
                                                    Required 
                                                    .false.
                                                ), 
                                            prinf_arg_1:
                                                (Variable 
                                                    3 
                                                    prinf_arg_1 
                                                    Unspecified 
                                                    () 
                                                    () 
                                                    Default 
                                                    (Integer 4 []) 
                                                    Source 
                                                    Public 
                                                    Required 
                                                    .false.
                                                ), 
                                            prinf_arg_2:
                                                (Variable 
                                                    3 
                                                    prinf_arg_2 
                                                    Unspecified 
                                                    () 
                                                    () 
                                                    Default 
                                                    (Integer 4 []) 
                                                    Source 
                                                    Public 
                                                    Required 
                                                    .false.
                                                )
                                            
                                        }) 
                                    prinf 
                                    [(Var 3 prinf_arg_0)
                                    (Var 3 prinf_arg_1)
                                    (Var 3 prinf_arg_2)] 
                                    [] 
                                    () 
                                    Source 
                                    Public 
                                    Interface 
                                    () 
                                    .false. 
                                    .false. 
                                    .false. 
                                    .false. 
                                    .false. 
                                    [] 
                                    [] 
                                    .false.
                                ), 
                            w:
                                (Variable 
                                    2 
                                    w 
                                    Unspecified 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            x:
                                (Variable 
                                    2 
                                    x 
                                    Unspecified 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                ), 
                            y:
                                (Variable 
                                    2 
                                    y 
                                    Unspecified 
                                    () 
                                    () 
                                    Default 
                                    (Integer 4 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    idd_frm 
                    [(Var 2 m)
                    (Var 2 n)
                    (Var 2 w)
                    (Var 2 x)
                    (Var 2 y)] 
                    [(SubroutineCall 
                        2 prinf 
                        () 
                        [((StringConstant 
                            "lw = *" 
                            (Character 1 6 () [])
                        ))
                        ((Var 2 lw))
                        ((IntegerConstant 1 (Integer 4 [])))] 
                        ()
                    )
                    (SubroutineCall 
                        2 prinf 
                        () 
                        [((StringConstant 
                            "16m+70 = *" 
                            (Character 1 10 () [])
                        ))
                        ((IntegerBinOp 
                            (IntegerBinOp 
                                (IntegerConstant 16 (Integer 4 [])) 
                                Mul 
                                (Var 2 m) 
                                (Integer 4 []) 
                                ()
                            ) 
                            Add 
                            (IntegerConstant 70 (Integer 4 [])) 
                            (Integer 4 []) 
                            ()
                        ))
                        ((IntegerConstant 1 (Integer 4 [])))] 
                        ()
                    )
                    (Return)] 
                    () 
                    Source 
                    Public 
                    Implementation 
                    () 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    .false. 
                    [] 
                    [] 
                    .false.
                )
            
        }) 
    []
)
